joomla/joomla-cms
# [4.0] Fix filter for workflows

Pull Request for Issue #24814

### Summary of Changes
The #__workflows table uses column name published. This causes several issues. One of them is that the ordering of workflows via state is not possible. 
Another isssue is that filters are not set, as described in the related issue. 


### Testing Instructions
Ordering: 
You must have at least two workflows. Set one of the unpublished and try ordering of the column "state"
Active Filters:
Or see test instruction in the related issue.

### Expected result
Columns are ordered new if possible
Active Filters are set


### Actual result
No ordering possible, active Filters are not set.


### Documentation Changes Required
no, it is a bug
